README.md

```diff
@@ -1935,6 +1935,19 @@ $ brew unlink trivy && brew uninstall trivy
 $ brew install aquasecurity/trivy/trivy
 ```
 
+### Error: FATAL	unable to initialize the cache: failed to create cache dir: mkdir /root/.cache: permission denied
+
+```
+$ docker run --rm -v $PWD:/root/.cache/  aquasec/trivy:0.9.2 --cache-dir /root/.cache image centos:7
+2020-07-29T15:02:54.435Z	FATAL	unable to initialize the cache: failed to create cache dir: mkdir /root/.cache: permission denied
+```
+
+Try:
+
+```
+$ docker run --rm -v $PWD:/root/.cache/  aquasec/trivy:0.9.2 --cache-dir root/.cache image centos:7
```

Comment:
This command doesn't store the cache in a host, right? You mount the host directory to `/root/.cache`, but it will be never used as you specify `/newfolder/root/.cache`. The reason a user mounts the volume is that they want to make the cache persistent.

So, `/tmp` looks better.

```suggestion
$ docker run --rm -v $PWD:/tmp/.cache/  aquasec/trivy:0.9.2 --cache-dir /tmp/.cache image centos:7
```

Reply:
@knqyf263 Thanks for the clarification, fixed 👍